BlueShift-iOS-SDK/BlueshiftIntegrationSwizzle.m

```diff
@@ -0,0 +1,206 @@
+            [self swizzleMethodWithClass:class originalSelector:originalSelector andSwizzledSelector:swizzledSelector];
+        }
+        
+        if ([uiApplicationDelegate respondsToSelector:@selector(userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:)]) {
+            SEL originalSelector = @selector(userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:);
+            SEL swizzledSelector = @selector(blueshift_swizzled_userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:);
+            [self swizzleMethodWithClass:class originalSelector:originalSelector andSwizzledSelector:swizzledSelector];
+        } else {
+            SEL originalSelector = @selector(userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:);
+            SEL swizzledSelector = @selector(blueshift_swizzled_no_userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:);
+            [self swizzleMethodWithClass:class originalSelector:originalSelector andSwizzledSelector:swizzledSelector];
+        }
+
+        if ([uiApplicationDelegate respondsToSelector:@selector(userNotificationCenter:willPresentNotification:withCompletionHandler:)]) {
+            SEL originalSelector = @selector(userNotificationCenter:willPresentNotification:withCompletionHandler:);
+            SEL swizzledSelector = @selector(blueshift_swizzled_userNotificationCenter:willPresentNotification:withCompletionHandler:);
+            [self swizzleMethodWithClass:class originalSelector:originalSelector andSwizzledSelector:swizzledSelector];
+        } else {
+            SEL originalSelector = @selector(userNotificationCenter:willPresentNotification:withCompletionHandler:);
+            SEL swizzledSelector = @selector(blueshift_swizzled_no_userNotificationCenter:willPresentNotification:withCompletionHandler:);
+            [self swizzleMethodWithClass:class originalSelector:originalSelector andSwizzledSelector:swizzledSelector];
+        }
+                
+        
+        if ([uiApplicationDelegate respondsToSelector:@selector(application:continueUserActivity:restorationHandler:)]) {
+            SEL originalSelector = @selector(application:continueUserActivity:restorationHandler:);
+            SEL swizzledSelector = @selector(blueshift_swizzled_application:continueUserActivity:restorationHandler:);
+            [self swizzleMethodWithClass:class originalSelector:originalSelector andSwizzledSelector:swizzledSelector];
+        } else {
+            SEL originalSelector = @selector(application:continueUserActivity:restorationHandler:);
+            SEL swizzledSelector = @selector(blueshift_swizzled_no_application:continueUserActivity:restorationHandler:);
+            [self swizzleMethodWithClass:class originalSelector:originalSelector andSwizzledSelector:swizzledSelector];
+        }
```

Comment:
this is a cosmetic change suggestion. Feel free to ignore, if the coding style doesn't match the standards.

since we have similar code for all the above `if else` blocks, does it make sense to add a method that accepts 3 params (say, selectorRespondsTo, selectorOriginal, selectorSwizzled) and takes care of doing the check and calling the `swizzleMethodWithClass` method with correct attributes?